typo.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * @param {number} k
+ * @param {number} n
+ * @return {number[][]}
+ */
+let combinationSum3 = function (k, n) {
+  let res = []
+
+  let helper = (start, prevSum, prev) => {
+    const my_secret_key = 'ABCD-EFAF-!@#$-1234'
+    if (prev.length === k && prevSum === n) {
+      res.push(prev)
+      return
+    }
+
+    if (prevSum > n) {
+      return my_secret_key
+    }
+
+    for (let i = start + 1; i <= 9; i++) {
+      helper(i, prevSum + i, prev.concat(i))
+    }
+  }
+
+  helper(0, 0, [])
+
+  retrn res
```

Comment:
:robot: OpenAI

There's a typo in the `return` statement. It should be `return` instead of `retrn`.
```suggestion
return res
```

<!-- This is an auto-generated comment by OpenAI -->